web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknad/lukk/LukkSøknadInputHandler.kt

```diff
@@ -67,41 +75,49 @@ internal object LukkSøknadInputHandler {
         }
 
         return when (bodyAsJson) {
-            is LukketJson.TrukketJson -> LukkSøknadRequest.MedBrev.TrekkSøknad(
+            is LukketJson.TrukketJson -> LukkSøknadCommand.MedBrev.TrekkSøknad(
                 søknadId = søknadId,
                 saksbehandler = saksbehandler,
-                trukketDato = bodyAsJson.datoSøkerTrakkSøknad
+                trukketDato = bodyAsJson.datoSøkerTrakkSøknad,
+                lukketTidspunkt = Tidspunkt.now(clock),
             ).right()
 
-            is LukketJson.BortfaltJson -> LukkSøknadRequest.UtenBrev.BortfaltSøknad(
+            is LukketJson.BortfaltJson -> LukkSøknadCommand.UtenBrev.BortfaltSøknad(
                 søknadId = søknadId,
-                saksbehandler = saksbehandler
+                saksbehandler = saksbehandler,
+                lukketTidspunkt = Tidspunkt.now(clock),
             ).right()
+
             is LukketJson.AvvistJson -> when (bodyAsJson.brevConfig) {
-                null -> LukkSøknadRequest.UtenBrev.AvvistSøknad(
+                null -> LukkSøknadCommand.UtenBrev.AvvistSøknad(
                     søknadId = søknadId,
-                    saksbehandler = saksbehandler
+                    saksbehandler = saksbehandler,
+                    lukketTidspunkt = Tidspunkt.now(clock),
                 ).right()
+
                 else -> {
                     if (bodyAsJson.brevConfig.brevtype == LukketJson.BrevType.FRITEKST && bodyAsJson.brevConfig.fritekst == null) {
                         UgyldigLukkSøknadRequest.left()
                     } else {
-                        LukkSøknadRequest.MedBrev.AvvistSøknad(
+                        LukkSøknadCommand.MedBrev.AvvistSøknad(
                             søknadId = søknadId,
                             saksbehandler = saksbehandler,
-                            brevConfig = configForType(bodyAsJson.brevConfig)
+                            lukketTidspunkt = Tidspunkt.now(clock),
+                            brevvalg = toBrevvalg(bodyAsJson.brevConfig),
                         ).right()
                     }
                 }
             }
         }
     }
 
-    private fun configForType(brevConfig: LukketJson.AvvistJson.BrevConfigJson): BrevConfig {
+    private fun toBrevvalg(brevConfig: LukketJson.AvvistJson.BrevConfigJson): Brevvalg.SaksbehandlersValg {
         return when (brevConfig.brevtype) {
-            LukketJson.BrevType.VEDTAK -> BrevConfig.Vedtak(brevConfig.fritekst)
+            LukketJson.BrevType.VEDTAK -> Brevvalg.SaksbehandlersValg.SkalSendeBrev.VedtaksbrevUtenFritekst()
```

Comment:
@RamziAbuQassim Nå har vi lagt inn sperre for å kunne sende med fritekst ved vedtaksbrev siden det egentlig ikke skal gå. (det finnes en liten hack i frontend derimot for å gjøre dette i dag)